legend-engine-language-pure-grammar/src/main/java/org/finos/legend/engine/language/pure/grammar/from/data/embedded/ModelStoreDataParseTreeWalker.java

```diff
@@ -65,9 +71,26 @@ public ModelStoreData visit(ModelStoreDataParserGrammar.DefinitionContext ctx)
         for (ModelStoreDataParserGrammar.TypeIndexedInstancesContext typeIndexedInstancesContext : ctx.typeIndexedInstances())
         {
             String fullPath = PureGrammarParserUtility.fromQualifiedName(typeIndexedInstancesContext.qualifiedName().packagePath() == null ? Collections.emptyList() : typeIndexedInstancesContext.qualifiedName().packagePath().identifier(), typeIndexedInstancesContext.qualifiedName().identifier());
-
-            ValueSpecification instances = collection(typeIndexedInstancesContext.instance().stream().map(this::visitInstance).collect(Collectors.toList()));
-
+            ValueSpecification instances = null;
+            if (typeIndexedInstancesContext.instance() != null)
+            {
+                instances = collection(typeIndexedInstancesContext.instance().stream().map(this::visitInstance).collect(Collectors.toList()));
+            }
+            if (typeIndexedInstancesContext.embeddedData() != null)
+            {
+                EmbeddedData embeddedData = HelperEmbeddedDataGrammarParser.parseEmbeddedData(typeIndexedInstancesContext.embeddedData(), this.walkerSourceInformation, extensions);
+                if (embeddedData instanceof DataElementReference)
+                {
+                    PackageableElementPtr ptr = new PackageableElementPtr();
+                    ptr.fullPath = ((DataElementReference) embeddedData).dataElement;
+                    PackageableElementPtr bindingPtr = new PackageableElementPtr();
+                    bindingPtr.fullPath = "default::" + fullPath + "::binding";
```

Comment:
lets add $ or something so that it doesn't conflict easily

Reply:
I believe this was missed while addressing

Reply:
view changes not showing up correctly but have changed that